### DISCUSSION TITLE
Create Job listener to only listen for specific job type

### DISCUSSION THREAD

**Discussion by MaxBrooks114** [quartznet/quartznet#1729] [Q&A]
I have several different jobs and each of those jobs has a listener which I use to chain another job. Is there any way to have a listener listen only to that job type without giving a group or key? eg. SampleJob has a SampleJobListener and SampleJobListener only fires its methods when something with SampleJob happens, regardless of key or group?

**Comment by lahma** [comment]
You could probably create a catch-all listener and check job type/attributes inside it?
